container/rkt/client.go

```diff
@@ -37,25 +39,83 @@ var (
 	once         sync.Once
 )
 
+type rktVersion struct {
+	*semver.Version
+}
+
+func (r rktVersion) Compare(other string) (int, error) {
```

Comment:
(optional) This is already implemented by "github.com/blang/semver".Version.Compare